### ISSUE TITLE
Implement util functions for alerts related UI components

### ISSUE THREAD

**Issue by ChengYanJin** [scality/metalk8s#3147]
**Component**: ui, monitoring

**Why this is needed**:
https://docs.google.com/document/d/1QrKJep8tQ31go7GV1YRGtfXEzg8Z6fcexTWoVCR2gCY/edit

**What should be done**:

**Implementation proposal** (strongly recommended):
Put in place the following utils functions: 

```js
getActiveAlerts(selectors: {[labelName: string]: string}): Promise<AlertList>

getHistoryAlerts(selectors: {[labelName: string]: string}, startDate: string, endDate?: string = 'now in iso format'): Promise<AlertList>

checkActiveAlertProvider(): Promise<"HEALTHY" | "CRITICAL">

checkPastAlertProvider(): Promise<{status: "HEALTHY" | "CRITICAL", retention: duration in s}> 
 => query /config to get retention

getStatus(alertList: AlertList, activeOn: string): "HEALTHY" | "WARNING" | "CRITICAL"

```

**Test plan**:
unit tests

**Comment by gdemonet** [comment]
> **Test plan**: unit tests

I think this needs more than unit testing. Since the utility methods really depend on the backend APIs which are deployed, I think we should at least test them in integration, if not end-to-end (likely simpler to setup). If these methods can be exposed through plain NodeJS, we could run them outside of a browser (not sure how hard this would be to implement).

**Comment by ChengYanJin** [comment]
Merged in #3149
